### PR TITLE
Annotation Integration

### PR DESCRIPTION
**Pull request by YmirKhang**
Changes proposed in this pull request:
-Annotation server configuration is added
-Annotation model for the backend is prepared
-Annotation support for the web application is created

### CODE REVIEW

**Comment by yigitozkavci** [comment]
Do you have any idea why might I be having this error? @emir350z 

image

File: Atlas/atlasAnnoServer/atlasAnnoServer/views.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        body = None
+        if 'body' in request.data:
+            body = request.data['body']
+
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        self.perform_create(serializer)
+
+        if (body is not None):
+            print("naber")
```
Comment (SaitTalhaNisanci) [inline Atlas/atlasAnnoServer/atlasAnnoServer/views.py]: dont forget to remove these

File: Atlas/atlasAnnoServer/atlasAnnoServer/views.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
+
+class annotationRetrieve(generics.ListAPIView):
+    serializer_class = annotation_serializer
+    lookup_field = target.IRI
+
+    def get_queryset(self):
+        query = self.kwargs.get('query')
+        print(query)
```
Comment (SaitTalhaNisanci) [inline Atlas/atlasAnnoServer/atlasAnnoServer/views.py]: remove this

File: Atlas/atlasAnnoServer/atlasAnnoServer/views.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        annots = annotation.objects.filter(target__IRI=query)
+        print(len(annots))
```
Comment (SaitTalhaNisanci) [inline Atlas/atlasAnnoServer/atlasAnnoServer/views.py]: remove

**Comment by bounaykut** [comment]
screenshot from 2017-12-27 12-01-11

I got an error like that when I click on the image on show content page.


And recommended items on the right side are not loaded. It prints "Loading" but never loads

**Comment by yigitozkavci** [comment]
@bounaykut Frontend isn't ready for annotations yet. It's in progress

**Review by yigitozkavci — APPROVED** [review]

**Pull request merged**